src/validation/email.js

```diff
@@ -0,0 +1,21 @@
+import isEmpty from './isEmpty';
+import isValidEmail from './isValidEmail';
+
+const validateEmail = ({ email }) => {
+  const errors = {};
+
+  email = !isEmpty(email) ? email : '';
+
+  if (isEmpty(email)) {
+    errors.email = 'Email is required';
+  } else {
+    !isValidEmail(email) ? errors.email = 'Email is invalid' : null;
```

Comment:
can you make this an `else if`? there is no point to the ternary operator on line 12.

Reply:
@conquext